ddp-workspace/projects/ddp-sdk/src/lib/models/analyticsActionTypes.ts

```diff
@@ -1,4 +1,4 @@
-export enum GoogleAnalytics {
+export enum AnalyticsActionTypes {
```

Comment:
We should consider removing `clickedCountMeIn` from here. This is more of a toolkit thing.

Reply:
Why? I think ROPES rely on the events which defined in the enum.

Reply:
Maybe CountMeIn is a CountMeInAnalyticsType toolkit? API calls just take strings so that would work. Making observation that everything else there is fairly generic and could be used by any other non-CMI project. This is the one exception.
Not a huge deal. Just a string, but mindset ought to be that SDK knows nothing about CMI.

Reply:
I think we need to leave it as is. SDK doesn't emit any events with this value, but I'd prefer to store all analytics constants in one place.